front-end/yarn.lock

```diff
@@ -3449,6 +3454,11 @@ acorn@^7.1.0:
   resolved "https://registry.yarnpkg.com/acorn/-/acorn-7.1.0.tgz#949d36f2c292535da602283586c2477c57eb2d6c"
   integrity sha512-kL5CuoXA/dgxlBbVrflsflzQ3PAas7RYZB52NOm/6839iVYJgKMJ3cQJD+t2i5+qFa8h3MDpEOJiS64E8JLnSQ==
 
+acorn@^7.1.1:
```

Comment:
That's silly that a tool to analyse bloated bundles is actually bloating it a little more with all these acorn versions.

Reply:
But its not being compiled to bundle with webpack. Its in devDependencies

Reply:
sure, I was half joking